rules/slack_rules/slack_app_access_expanded.py

```diff
@@ -27,7 +31,7 @@ def alert_context(event):
     prv_scopes = event.deep_get("details", "previous_scopes", default=[])
 
     context["scopes_added"] = [x for x in new_scopes if x not in prv_scopes]
-    context["scoped_removed"] = [x for x in prv_scopes if x not in new_scopes]
+    context["scopes_removed"] = [x for x in prv_scopes if x not in new_scopes]
 
     return context
 
```

Comment:
```suggestion
def get_scope_changes(event) -> dict[str, list[str]]:
    changes = {}

    new_scopes = event.deep_get("details", "new_scopes", default=[])
    prv_scopes = event.deep_get("details", "previous_scopes", default=[])

    changes["added"] = [x for x in new_scopes if x not in prv_scopes]
    changes["removed"] = [x for x in prv_scopes if x not in new_scopes]

    return changes

```
This is the proposed function